docs/design/shifu-framework/telemetry-secret.md

```diff
@@ -0,0 +1,92 @@
+# Telemetry Secret Loading Design
+
+## Introduction
+
+Telemetry Service is a standalone service, hereafter referred to as TSCenter. But the settings in TelemetryService CRD is loaded by Deviceshifu and sent to TSCenter. If the setting contains password, there will exist the problem of plaintext password in communication between Deviceshifu and TSCenter.
```

Comment:
let's call this as telemetry service always and be consist across the doc. avoid introduce new concept like "TSCenter"